unicodetools/data/ucd/11.0.0-Update/PropList.txt

```diff
@@ -1266,8 +1266,9 @@ AABB..AABC    ; Logical_Order_Exception # Lo   [2] TAI VIET VOWEL AUE..TAI VIET
 0387          ; Other_ID_Continue # Po       GREEK ANO TELEIA
 1369..1371    ; Other_ID_Continue # No   [9] ETHIOPIC DIGIT ONE..ETHIOPIC DIGIT NINE
 19DA          ; Other_ID_Continue # No       NEW TAI LUE THAM DIGIT ONE
+200C..200D    ; Other_ID_Continue # Cf   [2] ZERO WIDTH NON-JOINER..ZERO WIDTH JOINER
```

Comment:
This is in the wrong file path: You are changing the contents of Unicode 11. Changes for Unicode 15.1 need to go into .../data/ucd/dev/

Reply:
Ah, that explains why CI was not catching the things I expected it to catch.